### DISCUSSION TITLE
Sunset Progress for January 2023

### DISCUSSION THREAD

**Discussion by zoracon** [EFForg/https-everywhere#20213] [Sunset]
# Announcements 
- [x] Blog: https://www.eff.org/deeplinks/2021/09/https-actually-everywhere
- [x] Web Page explaining HTTPS Only Mode in browsers: https://www.eff.org/https-everywhere/set-https-default-your-browser
- [x] Extension: https://github.com/EFForg/https-everywhere/releases/tag/2022.5.24
- [x] Repo: https://github.com/EFForg/https-everywhere#readme

# Affiliated Websites that will need to be taken down
- [ ] https://atlas.eff.org/index.html
- [ ] https://www.https-rulesets.org
- [ ] https://www.https-rulesets.org/v1/
- [ ] https://www.https-rulesets.org/ddg/

- Archived information will still be at https://www.eff.org/https-everywhere.

# De-listing in Stores
- [x] https://addons.mozilla.org/
- [x] https://chrome.google.com/webstore/detail/https-everywhere/gcbommkclmclpchllfjekcdonpmejbdp
- [x] https://microsoftedge.microsoft.com/addons/detail/https-everywhere/fchjpkplmbeeeaaogdbhjbgbknjobohb
- [x] https://addons.opera.com/en/extensions/details/https-everywhere/

# Misc.

- [ ] Unpin from org repo
- [ ] Decomission private key for Chrome signing
- [ ] Decomission email list servs:

HTTPS-Everywhere | [HTTPS Everywhere Genral](https://lists.eff.org/mailman/listinfo/https-everywhere)
-- | --
HTTPS-Everywhere-Rules | [Rulesets](https://lists.eff.org/mailman/listinfo/https-everywhere-rule)
- [ ] Close MM channels

**Comment by KOLANICH** [comment]
I don't think that sunsetting the extension and delisting it from stores and removing the rulesets is a right move. They are still useful. We still cannot just blindly rewrite `http` to `https`. For example rulesets are used to bulk-rewrite links in files. And the non-enforcing mode of the extension is useful when dealing wjth people who just disable the extension wholly if it js in the enforcing mode. Also UX of browser's HTTPS-only mode is poor.

I think the extension and the datasets should not be "sunset".
